node/network/availability-distribution/src/tests/state.rs

```diff
@@ -204,7 +204,7 @@ impl TestState {
 					overseer_signal(update_tx.clone(), OverseerSignal::ActiveLeaves(update)).await;
 					// We need to give the subsystem a little time to do its job, otherwise it will
 					// cancel jobs as obsolete:
-					Delay::new(Duration::from_millis(20)).await;
+					Delay::new(Duration::from_millis(40)).await;
```

Comment:
why not something like 100ms? shouldn't slow down the tests much

Reply:
On my machine it works fine already with 3ms, so I thought 20 was already conservative. :laughing: ... anyhow, I am fine with 100 as well.

Reply:
that's only because your machine is not under heavy load with lots of PRs to test :)